jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="./test/e2e_node/jenkins/e2e-node-jenkins.sh"
```

Comment:
@ixdy I appended `./` here ( seems it's the issue? ) however https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-node-kubelet.sh#L22 works fine so I'm confused.. tell me if I'm doing things right here..